figures/metrics.py

```diff
@@ -1108,3 +1108,49 @@ def get_monthly_site_metrics(site, date_for=None, **kwargs):
         total_course_completions=total_course_completions,
         total_active_courses=total_active_courses,
     )
+
+
+def get_edly_monthly_site_metrics(site, date_for=None, **kwargs):
```

Comment:
Why we didn't write tests for it?

Reply:
The unit test is using mock_get_monthly_site_metrics for testing this feature, we have already updated the function in tests.